students/15th/team3/user/views.py

```diff
@@ -0,0 +1,46 @@
+from django.shortcuts import render
+import re
+import json
+from django.http     import JsonResponse
+from .models   import SignUp
+from django.views    import View
+
+class SignUpView(View):
+
+
+        def post(self, request):
+            data = json.loads(request.body)
+
+            data['email'] = data.get('email')
+            data['tel'] = data.get('tel')
+            data['name'] = data.get('name')
```

Comment:
- `data` 라는 변수는 `request.body` 를 파이썬 `dictionary` 로 변환하여 저장하는 변수인데, `data` 에서 꺼내온 값으로 `data`를 변형하고 있습니다.
- `data` 에서 꺼내온 값들을 아래처럼 따로 변수에 담아 사용한다면, 아래 코드들에서도 조건문들이 깔끔해집니다.
```suggestion
            data = json.loads(request.body)

            email = data.get('email')
            tel   = data.get('tel')
            name  = data.get('name')
```
- 그리고 여기서 `get()` 을 사용하신 이유를 설명해주세요.

Reply:
따로 변수를 지정해서 수정했습니다.
get() 을 쓴 이유는, 클라이언트에서 email, tel, name 의 값들이 전달되지 않으면, none 으로 받아 keyError 를  막기위해 사용했습니다!